mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -191,7 +194,8 @@ object CrossValidator extends MLReadable[CrossValidator] {
 
 /**
  * :: Experimental ::
- * Model from k-fold cross validation.
+ * Pipelines facilitate model selection by making it easy to tune an entire 
```

Comment:
I'm not sure if this is the best place to mention that - it seems like it doesn't belong in the CrossValidatorModel scaladoc.